.circleci/config.yml

```diff
@@ -156,3 +157,29 @@ jobs:
       - run: mv release/orca* artifacts
       - store_artifacts:
           path: artifacts
+
+  conda-build-and-upload:
+    docker:
+      - image: circleci/node:8.11.1-browsers
+    steps:
+      - restore_cache:
+          key: v1-source-{{ .Branch }}-{{ .Revision }}
+          keys:
+            - v1-source-{{ .Branch }}-
+            - v1-source-
+      - checkout
+      - save_cache:
+          key: v1-source-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - ".git"
+      - restore_cache:
+          key: v3-npm-{{ checksum "package-lock.json" }}
+          keys:
+            - v3-npm-
+
+      - run: wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
```

Comment:
We could probably cache this, but `conda-build-and-upload` runs in under 3 minutes, so I don't see the need :ok_hand: